src/libpriv/rpmostree-core.cxx

```diff
@@ -382,6 +382,17 @@ rpmostree_context_new_client (OstreeRepo   *repo)
   return self;
 }
 
+/* Low level API to create a context.  Avoid this in preference
+ * to creating a client or compose context unless necessary.
+ */
+RpmOstreeContext *
+rpmostree_context_new_base (OstreeRepo   *repo)
```

Comment:
Hmm, feels like `new_client` and `new_compose` should instead be using `new_base` and modifying on top, no? It's the same in the end, but makes the inheritance semantics clearer.

Reply:
So basically, we rename `rpmostree_context_new_client` to `rpmostree_context_new_base`, flip `is_system` to default to `FALSE` there, and this hunk would instead add `rpmostree_context_new_client` and flip `is_system` to `TRUE`.

Reply:
Yep, that is better - done.